rootfs/etc/profile.d/prompt.sh

```diff
@@ -102,22 +102,21 @@ function geodesic_prompt() {
 
 	if [ -n "$ASSUME_ROLE" ]; then
 		STATUS=${ASSUME_ROLE_ACTIVE_MARK}
-	else
-		STATUS=${ASSUME_ROLE_INACTIVE_MARK}
-	fi
-
-	if [ -n "${ASSUME_ROLE}" ]; then
 		ROLE_PROMPT="(${ASSUME_ROLE})"
 	else
+		STATUS=${ASSUME_ROLE_INACTIVE_MARK}
 		ROLE_PROMPT="(none)"
 	fi
 
+	local kops_root=""
+	[[ -n $KOPS_SSH_PRIVATE_KEY ]] && kops_root="+"
```

Comment:
This feels so incredibly bespoke/limited. Thinking `kops_root` should be generalized to incidate "presence of secrets" or something like that, of which `kops` is one kind.